src/pages/write-smart-contracts/billboard-tutorial.md

```diff
@@ -0,0 +1,281 @@
+---
+title: Billboard
+description: Learn how to store data on-chain and transfer STX tokens with Clarity
+duration: 30 minutes
+experience: intermediate
+tags:
+  - tutorial
+images:
+  large: /images/pages/billboard.svg
+---
+
+## Introduction
+
+This tutorial demonstrates how to transfer STX tokens and handle errors in Clarity by building a simple on-chain message
+store. Additionally, this tutorial provides a simple overview of testing a smart contract. This tutorial builds on
+concepts introduced in the [counter tutorial][], and uses [Clarinet][] to develop and test the smart contract.
+
+In this tutorial you will:
+
+- Set up a development environment with Clarinet
+- Define codes for error handling
+- Add a data storage variable with functions to get and set the variable
+- Add a STX transfer function within the variable setter
+- Develop a unit test to verify the contract works as expected
+
+The [final code for this tutorial][] is available in the Clarinet repository.
+
+## Prerequisites
+
+For this tutorial, you should have a local installation of [Clarinet][]. Refer to [Installing Clarinet][] for
+instructions on how to set up your local environment. You should also have a text editor or IDE to edit the Clarity
+smart contract.
+
+For developing the unit test, it's recommended that you have an IDE with Typescript support, such as
+[Visual Studio Code][].
+
+## Step 1: set up the project
+
+With Clarinet installed locally, open a new terminal window and create a new Clarinet project. Add a smart contract and
+an empty test file to the project:
+
+```sh
+clarinet new billboard-clarity && cd billboard-clarity
+clarinet contract new billboard
+```
+
+These commands create the necessary project structure and contracts for completing this tutorial. Remember that at
+any point during this tutorial you can use `clarinet check` to check the validity of your Clarity syntax.
```

Comment:
I'm seeing an error for the check command:

```
error: Found argument 'check' which wasn't expected, or isn't valid in this context
```

I pinged @lgalabru to figure out what the issue might be